OpenOversight/tests/conftest.py

```diff
@@ -81,17 +82,29 @@ def pick_salary():
     return random.randint(100, 100000000) / 100
 
 
+def generate_random_date(year, month):
```

Comment:
Hmmmmm randomness in tests seems bad. What about picking a random number once and then hardcoding that into the tests?

Reply:
Unless, if this is standard practice in our test suite already, then I guess it's fine to keep doing it.

Reply:
I think it's okay because a lot of the other methods in `conftest.py`, like `pick_birth_date()` or `pick_gender()`, use `random.randint()` or `random.choice()`.  Let me know if you disagree.

Reply:
It is true that we do this in a lot of places and I think it is actually a problem. I saw an issue with the randomness of our test-data just yesterday.
But given that it is already so widespread I guess it's fine to keep doing it for now. And if it's just the actual value randomness is not as bad as deciding if a field is None or not based on randomness.

Reply:
PR #756 offers an alternate implementation that might have better random distribution since it hashes the officer name: https://github.com/lucyparsons/OpenOversight/pull/756/files#diff-4eef8ece740b94822be839e43d28d7a7R42

Reply:
> I saw an issue with the randomness of our test-data just yesterday.

What was the problem you saw with test data randomness?  Is it something that we need to open an issue for?

I can use the implementation from PR #756, but I would also like to know more about the problems with our existing use of random test data.

Reply:
I will open an issue on the concrete problem I saw and probably add some ideas I have for moving forward